lighthouse-core/audits/critical-request-chains.js

```diff
@@ -11,7 +11,7 @@ const ComputedChains = require('../computed/critical-request-chains.js');
 
 const UIStrings = {
   /** Imperative title of a Lighthouse audit that tells the user to reduce the depth of critical network requests to enhance initial load of a page. Critical request chains are series of dependent network requests that are important for page rendering. For example, here's a 4-request-deep chain: The biglogo.jpg image is required, but is requested via the styles.css style code, which is requested by the initialize.js javascript, which is requested by the page's HTML. This is displayed in a list of audit titles that Lighthouse generates. */
-  title: 'Minimize Critical Requests Depth',
+  title: 'Minimize critical requests depth',
```

Comment:
Now that I'm looking at it like an imperative sentence, I'm not sure this is right...
```suggestion
  title: 'Minimize depth of critical requests',
```
or
```suggestion
  title: 'Avoid chaining critical requests',
```

perhaps?

Reply:
I think "chaining" conveys what's going on better.